docs/04-operation-guides/operations/08-install-uninstall-upgrade-kyma-module.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Install, uninstall and upgrade a Kyma module
+---
+
+The Kyma project is currently in the transition phase from classic to modular Kyma. Learn how to install, uninstall and upgrade a Kyma module. To see the list of all Kyma modules, go to [Overview](/docs/01-overview/README.md).
+
+## Install a Kyma module
+
+To install a module, deploy its module manager and apply the module configuration. See the available Kyma modules with quick installation steps and links to their GitHub repositories.
+
+### Keda
+
+```bash
+kubectl apply -f https://github.com/kyma-project/keda-manager/releases/latest/download/keda-manager.yaml
+kubectl apply -f https://github.com/kyma-project/keda-manager/releases/latest/download/keda-default-cr.yaml
+```
+
+For more details see, [`keda-manager`](https://github.com/kyma-project/keda-manager) repository in GitHub.
+
+### BTP Operator
+
+```bash
+kubectl apply -f https://github.com/kyma-project/btp-manager/releases/latest/download/btp-operator.yaml
+kubectl apply -f https://github.com/kyma-project/btp-manager/releases/latest/download/btpoperator-default-cr.yaml
+```
```

Comment:
@PK85, @pbochynski Should I use those above or:
```bash
kubectl apply -f https://github.com/kyma-project/btp-manager/releases/latest/download/rendered.yaml
kubectl apply -f https://raw.githubusercontent.com/kyma-project/btp-manager/main/config/samples/operator_v1alpha1_btpoperator.yaml
```
?

Reply:
For now yes, we will adjust that later to have both things under release

Reply:
Updated

Reply:
@mmitoraj 
Final version:
```
https://github.com/kyma-project/btp-manager/releases/latest/download/btp-manager.yaml
https://github.com/kyma-project/btp-manager/releases/latest/download/btp-operator-default-cr.yaml
```

Reply:
Please see: https://github.com/kyma-project/btp-manager/blob/074d3e1d8389bb80be2f78b09eb5c253c1d37bf1/README.md#installation

Step 3 explains that it will be in the warning state and what to do to make it running.